### PR TITLE
Initial rock64 health monitor script

### PR DESCRIPTION
**Pull request by pfeerick**
Elements not present on the rock64 removed, and cpu temperature scaling corrected.

Based on the pine64_health.sh health monitor for the pine64 by longsleep:
https://github.com/longsleep/build-pine64-image/blob/master/simpleimage/platform-scripts/pine64_health.sh

### CODE REVIEW

File: package/root/usr/local/sbin/rock64_health.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+
+if [ "$(id -u)" -ne "0" ]; then
```
Comment (ayufan) [inline package/root/usr/local/sbin/rock64_health.sh]: What of here requires root?

File: package/root/usr/local/sbin/rock64_health.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	cpu_frequency
+	cpu_count
+	scaling_govenor
+	soc_temp
+}
+
+
+WATCH=""
+for i in "$@"; do
+	case $i in
```
Comment (ayufan) [inline package/root/usr/local/sbin/rock64_health.sh]: `getopt`?

Comment (ayufan) [inline package/root/usr/local/sbin/rock64_health.sh]: http://wiki.bash-hackers.org/howto/getopts_tutorial

File: package/root/usr/local/sbin/rock64_health.sh (hunk 1 above)
Comment (pfeerick) [inline package/root/usr/local/sbin/rock64_health.sh]: Reading the CPU frequency

File: package/root/usr/local/sbin/rock64_health.sh (hunk 2 above)
Comment (pfeerick) [inline package/root/usr/local/sbin/rock64_health.sh]: I do not intend to change it. This is simply a minor modification of an existing script from the pine64, not a re-write.

**Pull request merged**